examples/energy_mng/src/main.rs

```diff
@@ -0,0 +1,38 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+#![cfg_attr(feature = "mcu-board-support", no_std)]
```

Comment:
I think you don't need to have that conditional
```suggestion
#![no_std]
```

Reply:
I add it because there is a future plan for a desktop version and maybe it can be combined in one project if it's possible. And it allows me to check how the stuff also look likes with the other backends.